.github/workflows/build.yml

```diff
@@ -103,12 +113,19 @@ jobs:
           cp ./arm64-osx.cmake ./community/universal-osx.cmake
           sed -i '' 's/^set(VCPKG_OSX_ARCHITECTURES.*$/set(VCPKG_OSX_ARCHITECTURES "arm64;x86_64")/' ./community/universal-osx.cmake
 
+      - name: Import macOS code signing certificate
+        if: ${{ matrix.os.preset == 'mac' && env.MACOS_SIGNING == 'true' }}
+        uses: apple-actions/import-codesign-certs@v3
+        with:
+          p12-file-base64: ${{ secrets.MACOS_SIGNING_CERTIFICATE_P12 }}
+          p12-password: ${{ secrets.MACOS_SIGNING_CERTIFICATE_P12_PASSWORD }}
+
       - name: Configure CMake
         if: ${{ matrix.os.preset != 'linux-cross-arm64' }}
         env:
           CC: ${{ matrix.os.cc }}
           CXX: ${{ matrix.os.cxx }}
-        run: cmake --preset ${{ matrix.os.preset }} -DBUILD_TESTING=ON -DENABLE_LOGGER=ON -DFORCE_PORTABLE_INSTALL=ON -DBUILD_EDITOR=ON -DUSE_EXTERNAL_PLOG=ON
+        run: cmake --preset ${{ matrix.os.preset }} -DCODESIGN_IDENTITY=${{ secrets.MACOS_SIGNING_IDENTITY }} -DBUILD_TESTING=ON -DENABLE_LOGGER=ON -DFORCE_PORTABLE_INSTALL=ON -DBUILD_EDITOR=ON -DUSE_EXTERNAL_PLOG=ON
```

Comment:
The `CODESIGN_IDENTITY` CMake cache variable should be declared as such in the top-level `CMakeLists.txt`: `set(CODESIGN_IDENTITY ... CACHE STRING ...)` like we do for `USE_VCPKG` for example. It should also be documented in `BUILD.md`

Reply:
Oooo, maybe not fixed.  In fact, the build is failing.  Sad.

Reply:
I tried futzing with it but couldn't get it working.  I'll try again tomorrow.

Reply:
I'll look into it tonight and try to help you out

Reply:
see suggestions below

Reply:
Thanks for the help!  Fixed.